src/openms/source/ANALYSIS/MAPMATCHING/TransformationDescription.cpp

```diff
@@ -184,4 +184,68 @@ namespace OpenMS
     }
   }
 
+  void TransformationDescription::getDeviations(vector<double>& diffs, 
```

Comment:
Yes, I stuck with the way that is currently prevalent in OpenMS. However, when we switch to C++11/14, I think we should update our coding conventions to say: "For new code, return complex types by value rather than as a reference parameter..." (or something like that).